server/api/index.js

```diff
@@ -1,10 +1,12 @@
 const express = require('express');
 const users = require('./routes/users');
 const auth = require('./routes/auth');
+const subscriptions = require('./routes/subscriptions');
 
 const router = express.Router();
 
 router.use('/api/users', users);
 router.use('/auth', auth);
+router.use('/users/:userId/subscriptions', subscriptions);
```

Comment:
we're probably gonna wanna be consistent with naming conventions here as well. so it maybe makes sense to change the `/api/users` to just `/users`. But since subscriptions lives off of users, we can approach this one of many ways:

(1) either just use a `/subscriptions` route and attach the `userId` to the request itself to check for it
(2) move the subscription routing inside the users file

^^ It may make sense to go over both options on a call to clear this up